process-manager/process-manager.js

```diff
@@ -26,6 +26,7 @@ ProcessManager.prototype.start = function(cb) {
       return cb(err);
     }
 
+    process.env.PORT = '3001';
     pm.process = spawn(process.execPath, args, {
```

Comment:
there is an env: option to spawn(), use that: make sure to `util._extend(process.env, {PORT: 3001})`